### ISSUE TITLE
VS Code platform.io IDE usage

### ISSUE THREAD

**Issue by progl0r** [Torxgewinde/Firebeetle-2-ESP32-E#1]
Hey Torxgewinde,

thanks for the nicely done coding, very interesting application. My question is actually not related to the code itself. I guess you compiled and build the binaries with the Arduino IDE? Im working with VS code and platform.io but since I only find the Firebeetle (based on the ESP32, not E) board definitions in their database, I wonder if I could use the "old" board definition to flash your code. Do you have any information about that?

Best
Progl0r

**Comment by Torxgewinde** [comment]
Hello ProgL0r,
Thank you. Yes, the *.ino files can be compiled with Arduino and PlatformIO. Also your guess is correct: I just used the predecessor Firebeetle ESP32 as board. Just a short while ago (=10 days as of now) the [Arduino ESP32 released a new version](https://github.com/espressif/arduino-esp32/releases/tag/2.0.0) and I did not check yet if the Firebeetle 2 ESP32-E is now included. Practically, the difference between previous Firebeetle ESP32 (SKU:DFR0478) and ESP32-E (SKU:DFR0654) did not matter, because I looked up the GPIO / Pin definitions from the DFRobot [schematic](https://dfimg.dfrobot.com/nobody/wiki/fd28d987619c16281bdc4f40990e5a1c.PDF) and [Wiki](https://wiki.dfrobot.com/FireBeetle_Board_ESP32_E_SKU_DFR0654) and did not use the Pin numbers like D\<number\>. The binaries the compiler is generating for ESP32 seem to run on ESP32-E unchanged. Obviously, I recommend to use the correct target once it is selectable to avoid obscure issues as it still might be a little bit different. If you are curious, the DFRobot specific changes are available [here](http://download.dfrobot.top/FireBeetle/). You can download the *.zip file and see that there is a file ```0.2.1/variants/esp32-e/pins_arduino.h``` that is just a pin mapping that can be avoided if you just use the GPIO numbers according to the pinout provided by DFRobot. The ```02.1./boards.txt``` contains the compile options for both variants, but without using things like PSRAM etc. it does also not cause practical issues just selecting the "old" ESP32 based Firebeetle.

TL;DR: For this project selecting the Firebeetle ESP32 works. If ESP32-E is available use that.

**Comment by Torxgewinde** [comment]
There is a [matching issue at PlatformIO](https://github.com/platformio/platform-espressif32/issues/619). We will see when they will address it. Closing the issue here, because it has been answered.
